drivers/gles3/storage/material_storage.cpp

```diff
@@ -2285,6 +2285,58 @@ GLuint MaterialStorage::global_shader_uniforms_get_uniform_buffer() const {
 	return global_shader_uniforms.buffer;
 }
 
+int32_t MaterialStorage::global_shader_uniforms_unit_variable_allocate() {
```

Comment:
We should add some comments above this function. Essentially something like "Unit variable allocate is an alternative way of allocating global shader uniforms. It returns a single variable that can be used and its position in the global shader uniform buffer. This is intended to be used for debug drawing purposes and should not be used for anything substantial."